Cargo.lock

```diff
@@ -3684,6 +3679,11 @@ dependencies = [
  "xml-rs 0.7.0 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[patch.unused]]
+name = "ring"
+version = "0.12.1"
+source = "git+https://github.com/paritytech/ring#bae475e9f7ea7dd4ae671bef4b576089a9b06731"
```

Comment:
what is `patch.unused` property and how was it created? I've never seen this entry in `Cargo.lock` files